README.md

```diff
@@ -8,7 +8,7 @@ Open the included `hurricane-irma.Rproj`
 
 If you don't have it, install [`vizlab`](https://github.com/USGS-VIZLAB/vizlab)
 
-Check to make sure you have required packages listed in `viz.yml` installed.
+Check to make sure you have required packages listed in `viz.yml` installed. To get the `svglite` package, you will need to install it from GitHub using `devtools::install_github("jread-usgs/svglite@svgnano")`. For Windows users, you might see an error similar to `Error: running command '"C:/PROGRA~1/R/R-34~1.1/bin/x64/R" --no-site-file --no-environ --no-save --no-restore --quiet CMD config CC' had status 127`. If this is the case, clone [Jordan Read's repository](https://github.com/jread-usgs/svglite) and build the package locally.
```

Comment:
vizlab spells it `yaml`... is there a standard for this anywhere?

Reply:
Hmmm not that I know of